### PR TITLE
Feat: define Jekyll collection for Resources

### LINKED ISSUE #104: Resources Jekyll collection
Based on this sheet: https://docs.google.com/spreadsheets/d/1xJpgr8JBG9TgZF2mHB8Vjnhu453No4dHoNjFIhb0FQw/edit#gid=0

See `Column C - Hosted on (calitp.org)` and filter to `Resources Page` for the list to use for this collection.

- `category`: values in `Column B`
- `tags`: values in `Column D`
- `title`: values in `Column F`
- `asset`: values in `Column G`
- `date`

## Tasks

- [x] define collection in Jekyll
- [x] define collection items based on existing Resources (see above)
- [x] define template to render list of Resources

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #104 

Like for Press, styling of this landing page is left for #105

### CODE REVIEW

File: src/_resources/case-study-cash-app.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+date: 2022-11-01
+title: |-
+  Cash App Case Study: Instead of using cash, Monterey & Sacramento riders are choosing to tap the Cash App Card,
+  a free Visa debit card
+asset: Cal-ITP.CashApp.CaseStudy.pdf
```
Comment (machikoyasuda) [inline src/_resources/case-study-cash-app.md:6]: This link is interesting.. actually doesn't link to the full case study, just a one-pager: https://deploy-preview-144--cal-itp-website.netlify.app/assets/Cal-ITP.CashApp.CaseStudy.pdf

Is that the intent? 

Also, this  headline is sentence-case but the rest are title-case.

File: src/_resources/fact-sheet-gtfs-feed.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+date: 2021-05-01
+title: "Fact sheet: Why do I need a standard feed?"
```
Comment (machikoyasuda) [inline src/_resources/fact-sheet-gtfs-feed.md:3]: Fact Sheet

File: src/_resources/slide-deck-understanding-merchant-service-charges.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+date: 2023-03-01
+title: Understanding "Merchant Service Charges"
```
Comment (machikoyasuda) [inline src/_resources/slide-deck-understanding-merchant-service-charges.md:3]:
```suggestion
title: Understanding ‘Merchant Service Charges’
```

Comment (machikoyasuda) [reply src/_resources/slide-deck-understanding-merchant-service-charges.md:3]: smart quotes 😄

File: src/_resources/fact-sheet-automated-transit-discounts.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+date: 2021-05-01
+title: "Fact sheet: Automated Transit Discounts"
```
Comment (machikoyasuda) [inline src/_resources/fact-sheet-automated-transit-discounts.md:3]:
```suggestions
title: "Fact Sheet: Automated Transit Discounts"
```

File: src/_resources/fact-sheet-gtfs-feed.md (hunk 2 above)
Comment (machikoyasuda) [reply src/_resources/fact-sheet-gtfs-feed.md:3]:
```suggestion
title: "Fact Sheet: Why do I need a standard feed?"
```

**Review by machikoyasuda — APPROVED** [review]

**Review by machikoyasuda — CHANGES_REQUESTED** [review]
LGTM - just very small copy formatting changes, mentioned above as Suggestions

File: src/_resources/case-study-cash-app.md (hunk 1 above)
Comment (thekaveman) [reply src/_resources/case-study-cash-app.md:6]: > doesn't link to the full case study, just a one-pager

Yeah we have 2 versions, I'm not sure why / which one is used for what audience. But the [Resources catalog](https://docs.google.com/spreadsheets/d/1xJpgr8JBG9TgZF2mHB8Vjnhu453No4dHoNjFIhb0FQw/edit#gid=0) pointed to this bitly: https://bit.ly/3RWIcEP?r=lp which points to the single-page version.

> Also, this headline is sentence-case but the rest are title-case

Yeah I just copied directly from the Catalog entry?

@mrose914 thoughts on the above questions?

Comment (machikoyasuda) [reply src/_resources/case-study-cash-app.md:6]: Ah ~ Even in the Google Drive, there's only a _draft_ of the full case study. The one pager is fine for this PR.

Comment (machikoyasuda) [reply src/_resources/case-study-cash-app.md:6]: Marking this as resolved for the time being. @mrose914 If we're allowed/want to put the actual case study up, we can do it in a subsequent Issue and Pull Request.

**Review by machikoyasuda — APPROVED** [review]

**Pull request merged**